### DISCUSSION TITLE
Video Preview not working in admin UI

### DISCUSSION THREAD

**Discussion by navneetshukla9** [pimcore/pimcore#14687] [Q&A / Support]
In the assets section when I upload any video in any format it does not show its preview and keeps showing the loading gif.

Did anyone fix this before? please help here.

**Comment by kingjia90** [comment]
Please make sure the cronjob are properly set
https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html#page_5-Maintenance-Cron-Job
or try run `bin/console messenger:consume pimcore_core pimcore_maintenance pimcore_image_optimize pimcore_asset_update` in console

**Comment by navneetshukla9** [comment]
@kingjia90 Cron is set up and when I ran the command you sent it went fine the message I see is below but still the issue is same. Its not show preview nor create thumbnails of video

04:09:20 INFO      [messenger] Received message Pimcore\Messenger\SearchBackendMessage ["class" => "Pimcore\Messenger\SearchBackendMessage"]
04:09:20 INFO      [messenger] Message Pimcore\Messenger\SearchBackendMessage handled by Pimcore\Messenger\Handler\SearchBackendHandler::__invoke ["class" => "Pimcore\Messenger\SearchBackendMessage","handler" => "Pimcore\Messenger\Handler\SearchBackendHandler::__invoke"]
04:09:23 INFO      [messenger] Received message Pimcore\Messenger\SearchBackendMessage ["class" => "Pimcore\Messenger\SearchBackendMessage"]
04:09:23 INFO      [messenger] Message Pimcore\Messenger\SearchBackendMessage handled by Pimcore\Messenger\Handler\SearchBackendHandler::__invoke ["class" => "Pimcore\Messenger\SearchBackendMessage","handler" => "Pimcore\Messenger\Handler\SearchBackendHandler::__invoke"]
04:09:25 INFO      [messenger] Received message Pimcore\Messenger\SearchBackendMessage ["class" => "Pimcore\Messenger\SearchBackendMessage"]
04:09:25 INFO      [messenger] Message Pimcore\Messenger\SearchBackendMessage handled by Pimcore\Messenger\Handler\SearchBackendHandler::__invoke ["class" => "Pimcore\Messenger\SearchBackendMessage","handler" => "Pimcore\Messenger\Handler\SearchBackendHandler::__invoke"]

**Comment by navneetshukla9** [comment]
I have checked more and even images preview are not showing so none of the assets are showing its preview
